igrill.py

```diff
@@ -150,12 +157,21 @@ class IGrillV3Peripheral(IDevicePeripheral):
     def __init__(self, address, name='igrill_v3', num_probes=4):
         logging.debug("Created new device with name {}".format(name))
         IDevicePeripheral.__init__(self, address, name, num_probes)
+        
+class Pulse2000(IDevicePeripheral):
+    """
+    Specialization of iDevice peripheral for the Weber Pulse 2000
+    """
 
+    def __init__(self, address, name='pulse_2000', num_probes=4):
+        logging.debug("Created new device with name {}".format(name))
+        IDevicePeripheral.__init__(self, address, name, num_probes)
```

Comment:
if we add heating_element to IDevicePeripheral.__init__ as an argument with default value None, we can specify here when we initialize it, if the device should report in this.